pkg/handler/eventhandler_test.go

```diff
@@ -281,20 +277,16 @@ var _ = Describe("Eventhandler", func() {
 				Expect(q.Len()).To(Equal(4))
 
 				i, _ := q.Get()
-				Expect(i).To(Equal(reconcile.Request{
-					NamespacedName: types.NamespacedName{Namespace: "foo", Name: "baz-bar"}}))
+				Expect(i).To(Equal(reconcile.Request{NamespacedName: types.NamespacedName{Namespace: "foo", Name: "baz-bar"}}))
```

Comment:
What about changing this tests to not give the new pod a new name (which can not happen anyways with a real apiserver) so it tests the deduplication?

Reply:
Yeah, lets do that